### PR TITLE
New AMP homepage

### PR DESCRIPTION
**Pull request by markulrich**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-  "version": "0.2.0",
+  "version": "0.3.0",
   "scripts": {
-    "start": "react-scripts-ts start",
-    "build": "react-scripts-ts build",
-    "test": "react-scripts-ts test --env=jsdom",
+    "start": "yarn pre-build && react-scripts-ts start",
+    "build": "yarn pre-build && react-scripts-ts build",
+    "test": "yarn pre-build && react-scripts-ts test --env=jsdom",
+    "pre-build": "cp -r static public && static-i18n -o public -l en locales",
```
Comment (markulrich) [inline package.json]: A lot of this pre-build/post-build crap could be cleaned up if we run eject and then do things ourselves. Also need to make yarn start have hot-reload, I developed this by doing `serve -s locales` which was pretty lame.

However, not sure it makes sense to run eject any time soon given our mobile-first focus. At least wait for a couple upgrade cycles, might be nice to get whatever benefits come out of `react-scripts-ts 3.0`

File: locales/en.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "every_life": "Every life has value",
+  "invite_only": "Invite Only",
+  "already_member": "Already a member?",
+  "login": "Login",
+  "prosperity": "Prosperity",
+  "proesperity-deets": "We want to end extreme poverty with the Raha basic income",
+  "trust": "Trust",
+  "trust-deets": "Unique identity network of trusted individuals for the sharing economy",
+  "free": "Free",
+  "free-deets": "Raha is for everyone with no cost to join and voluntary transaction fees"
```
Comment (rahulgi) [inline locales/en.json]: as much as I like it, let's just call it "details" instead of "deets" lol.

File: locales/en.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "raha_network": "Raha.io Network",
+  "every_life": "Every life has value",
+  "invite_only": "Invite Only",
+  "already_member": "Already a member?",
+  "login": "Login",
+  "prosperity": "Prosperity",
+  "proesperity-deets": "We want to end extreme poverty with the Raha basic income",
```
Comment (rahulgi) [inline locales/en.json]: "prosperity" is spelled wrong here.

File: locales/en.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  "trust": "Trust",
+  "trust-deets": "Unique identity network of trusted individuals for the sharing economy",
```
Comment (rahulgi) [inline locales/en.json]: can we consistently use underscores, please?

File: package.json [hunk 5]
```diff
@@ -1,17 +1,18 @@
 {
   "name": "@raha.io/website",
-  "version": "0.2.0",
+  "version": "0.3.0",
   "scripts": {
-    "start": "react-scripts-ts start",
-    "build": "react-scripts-ts build",
-    "test": "react-scripts-ts test --env=jsdom",
+    "start": "yarn pre-build && react-scripts-ts start",
+    "build": "yarn pre-build && react-scripts-ts build",
```
Comment (rahulgi) [inline package.json]: Can we just run all of pre, build, and post here? No one seems to be calling `yarn build` on its own.

**Comment by markulrich** [comment]
AMP development moved to raha-homepage, closing this and addressing @rahulgi's comments at https://github.com/rahafoundation/raha-homepage/pull/1/files

**Pull request closed without merging**